### PR TITLE
Update page in APIG upgrade section “upgrade_steps_oneversion”

### PR DESCRIPTION
**Pull request by markduig-ax**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by cgardineraxway** [comment]
I have pinged @markduig-ax directly with some best practices when creating a PR. This PR is pending changes from @markduig-ax

File: content/en/docs/apim_installation/apigw_upgrade/upgrade_steps_oneversion.md [hunk 1]
```diff
@@ -256,3 +253,11 @@ Updating API Manager is now carried out during the application of the latest API
 * Policy Studio project upgrades. Importing an existing API Manager Policy Studio project will upgrade API Manager. The upgrade is also applied when creating a new project from an existing `fed` file.
 * API Manager `.fed` files can be upgraded using the [upgradeconfig](/docs/apim_installation/apigw_upgrade/upgrade_analytics#upgradeconfig-options) script.
 * The [projupgrade](/docs/apim_reference/devopstools_ref#projupgrade-command-options) script will apply API Manager updates to any existing projects.
+
+## Restoring from Backup
+
+When running the ./update_apigw.sh script users have the option of specifying a backup directory with specifying the  --backup_dir option.  This will create a copy of your existing installation directory, if there is a problem with the update then you will be able to use the content of the backup to restore to the point before starting the update.  Note: this will not back up the data in Cassandra or any of the external databases or storage.
```
Comment (cgardineraxway) [inline content/en/docs/apim_installation/apigw_upgrade/upgrade_steps_oneversion.md:259]: - change **with specifying the  --backup_dir option** to _by specifying the --backup_dir option_

File: content/en/docs/apim_installation/apigw_upgrade/upgrade_steps_oneversion.md [hunk 2]
```diff
@@ -256,3 +253,11 @@ Updating API Manager is now carried out during the application of the latest API
...
 * API Manager `.fed` files can be upgraded using the [upgradeconfig](/docs/apim_installation/apigw_upgrade/upgrade_analytics#upgradeconfig-options) script.
 * The [projupgrade](/docs/apim_reference/devopstools_ref#projupgrade-command-options) script will apply API Manager updates to any existing projects.
+
+## Restoring from Backup
+
+When running the ./update_apigw.sh script users have the option of specifying a backup directory with specifying the  --backup_dir option.  This will create a copy of your existing installation directory, if there is a problem with the update then you will be able to use the content of the backup to restore to the point before starting the update.  Note: this will not back up the data in Cassandra or any of the external databases or storage.  
+
+If there has a been significant changes to the configuration since the backup (for example changing the group configuration passphrase) you may not be to restore to a point where the external data will be compatible with the backup.
```
Comment (cgardineraxway) [inline content/en/docs/apim_installation/apigw_upgrade/upgrade_steps_oneversion.md:261]: - **If there has a been significant changes** remove the a
- **you may not be to restore** change to _you may not be able to_

File: content/en/docs/apim_installation/apigw_upgrade/upgrade_steps_oneversion.md [hunk 3]
```diff
@@ -256,3 +253,11 @@ Updating API Manager is now carried out during the application of the latest API
...
+
+Note: if you are updating to March2020 version or after you are recommended to run the kps re-encrypt script if you run this you will not be able to restore to a previous backup as some of the data will not be able to read by the older configuration.
```
Comment (cgardineraxway) [inline content/en/docs/apim_installation/apigw_upgrade/upgrade_steps_oneversion.md:263]: - Remove the Note:

- **kps re-encrypt script if** full stop after script.
- **some of the data will not be able to read by the older configuration.** change to _some of the data will not be readable by the older configuration._

**Review by cgardineraxway — CHANGES_REQUESTED** [review]

**Review by cgardineraxway — CHANGES_REQUESTED** [review]

**Comment by andreamussap** [comment]
@markduig-ax I can't perform my review until you address and **close** the open comments. Thanks

**Comment by markduig-ax** [comment]
Duplicate. Closing

**Pull request closed without merging**